pkg/storegateway/indexcache/tracing.go

```diff
@@ -32,11 +33,19 @@ func (t *TracingIndexCache) StorePostings(ctx context.Context, userID string, bl
 }
 
 func (t *TracingIndexCache) FetchMultiPostings(ctx context.Context, userID string, blockID ulid.ULID, keys []labels.Label) (hits map[labels.Label][]byte, misses []labels.Label) {
+	t0 := time.Now()
 	hits, misses = t.c.FetchMultiPostings(ctx, userID, blockID, keys)
 
 	spanLogger := spanlogger.FromContext(ctx, t.logger)
-	level.Debug(spanLogger).Log("msg", "cache_fetch_postings", "requested keys", len(keys), "cache hits", len(hits), "cache misses", len(misses), "user_id", userID)
-
+	level.Debug(spanLogger).Log(
+		"msg", "IndexCache.FetchMultiPostings",
```

Comment:
Just wondering if the prefix should be `TracingIndexCache` in the messages, considering the type name?

Reply:
I think that's something that is implicit, since the TracingIndexCache is the one that traces the IndexCache interface.